tests/assets/recipes/baked/daylight-factor.yaml

```diff
@@ -1,5 +1,7 @@
+type: BakedRecipe
```

Comment:
Is this "BakedRecipe" a new class type or meant to be "Recipe"?

Reply:
BakeRecipe is different from Recipe. You can think about it as the compiled version of the recipe. You will still have a normal recipe that you can work with.

Reply:
It is correct in the regular recipe.